log4j-extensions/pom.xml

```diff
@@ -31,6 +31,7 @@
     </properties>
 
     <artifactId>confluent-log4j-extensions</artifactId>
+    <groupId>io.confluent.logging</groupId>
```

Comment:
this is ok as long as this was merged for 5.2.x. if it was earlier, this would only be ok in major version bump.

real e2e validation is to run full packaging against 5.2.x but this PR branch for common. that should cover smoke tests and also provide testable artifacts if you want to do any manual validation.

Also, worth pointing out that you should be able to exclude via artifact and not just group, e.g. see https://github.com/confluentinc/schema-registry/blob/master/package-schema-registry/src/assembly/package.xml#L54 in schema reg. Not sure if there is a reason for this approach other than a) no compatibility issue and b) possibility of future additions that can be caught proactively.

Reply:
Yeah, these artifacts were introduced in 5.2.

> real e2e validation is to run full packaging against 5.2.x but this PR branch for common. that should cover smoke tests and also provide testable artifacts if you want to do any manual validation.

How do I do that?

> Also, worth pointing out that you should be able to exclude via artifact and not just group, e.g. see https://github.com/confluentinc/schema-registry/blob/master/package-schema-registry/src/assembly/package.xml#L54 in schema reg. Not sure if there is a reason for this approach other than a) no compatibility issue and b) possibility of future additions that can be caught proactively.

(b) was the big reason. The convention here seems to be that stuff packaged in common uses the group io.confluent with the artifact prefix common-. So in theory sticking to that convention will avoid future pain down the road.

Reply:
To run e2e, create a packaging branch where you override the branch for common by adding a `common_BRANCH=...` setting to `settings.sh`. If the change is only in your fork, you'll also need to adjust the `COMMON_REPO` setting. From there, a PR on the packaging branch containing those changes would run a full build as usual.